attributes/default.rb

```diff
@@ -157,6 +168,8 @@
 default['cfncluster']['cfn_node_type'] = nil
 default['cfncluster']['cfn_master'] = nil
 default['cfncluster']['cfn_cluster_user'] = 'ec2-user'
+default['cfncluster']['cfn_fsx_shared_dir'] = "NONE"
```

Comment:
minor: single apex

Reply:
Changed, I also changed `cfn_raid_parameters` to match.